index.js

```diff
@@ -580,23 +577,35 @@ RingPop.prototype.onMemberSuspect = function onMemberSuspect(member) {
 
 RingPop.prototype.onMembershipUpdated = function onMembershipUpdated(updates) {
     var self = this;
+    var membershipChanged = false;
+    var ringChanged = false;
 
     updates.forEach(function(update) {
         if (update.type === 'alive') {
             self.onMemberAlive(update);
+            ringChanged |= membershipChanged |= true;
```

Comment:
`|=` is for doing bit operations on numbers. It's weird to use the "coercion" feature of javascript for this.

We should use the long form. i.e. `ringChanged = true; membershipChanged = true;`